admin/category_product_listing.php

```diff
@@ -184,7 +190,7 @@
       zen_redirect(zen_href_link(FILENAME_CATEGORY_PRODUCT_LISTING, 'cPath=' . $cPath));
       break;
     case 'move_category_confirm':
-      if (isset($_POST['categories_id']) && ($_POST['categories_id'] != $_POST['move_to_category_id'])) {
+      if (isset($_POST['categories_id']) && ($_POST['categories_id'] !== $_POST['move_to_category_id'])) {
```

Comment:
Strict comparison will fail, since the admin sanitization has cast 'categories_id' to an int, but not 'move_to_category_id'.